app/res/layout/fragment_connect_learning_progress.xml

```diff
@@ -153,15 +154,27 @@
                         android:background="@color/connect_blue_color_10"
                         android:padding="16dp">
 
+                        <org.commcare.views.connect.connecttextview.ConnectMediumTextView
```

Comment:
Oops! We decided [here](https://dimagi.atlassian.net/browse/QA-7221?focusedCommentId=354783) to leave the extra "Progress" line out since it's redundant and can cause real estate issues.